jsonschema-core.xml

```diff
@@ -1483,27 +1483,22 @@
                         <t>
                             Together with "$dynamicAnchor", "$dynamicRef" implements a cooperative
                             extension mechanism that is primarily useful with recursive schemas
-                            (schemas that reference themselves).  Both the extension point and the
-                            runtime-determined extension target are defined with "$dynamicAnchor",
-                            and only exhibit runtime dynamic behavior when referenced with
-                            "$dynamicRef".
+                            (schemas that reference themselves).  The extension point is defined with
+                            "$dynamicAnchor" and only exhibits runtime dynamic behavior when referenced
+                            with "$dynamicRef".
                         </t>
                         <t>
-                            The value of the "$dynamicRef" property MUST be a string which is
-                            a IRI-Reference.  Resolved against the current IRI base, it produces
-                            the IRI used as the starting point for runtime resolution.  This initial
-                            resolution is safe to perform on schema load.
+                            The value of the "$dynamicRef" property MUST be a string which is a
+                            IRI-Reference that contains a valid <xref target="anchor">plain name
+                            fragment</xref>.  Resolved against the current IRI base, it indicates
+                            the schema resource used as the starting point for runtime resolution.
```

Comment:
This is unclear. So the $dynamicRef value gets resolved against the base IRI and we get an absolute IRI. This absolute IRI can be resolved to a schema resource (= object?). What is it a starting point for?

Instead of "starting point" , is it accurate to say that the dynamic resolution should use this initially resolved schema as the *default* product of the $dynamicRef lookup in cases when there is no matching $dynamicAnchor on the dynamic path? If yes, then please use this wording, would be much more understandable.

Reply:
I think this is more clear with a little more familiarity with some of the newer concepts in the spec. Here are a few sections you should (re-)read.

> A JSON Schema resource is a schema which is canonically identified by an absolute URI. - https://json-schema.org/draft/2020-12/json-schema-core.html#rfc.section.4.3.5

> The outermost dynamic scope is the schema object at which processing begins - https://json-schema.org/draft/2020-12/json-schema-core.html#rfc.section.7.1

I always thought "starting point" doesn't make sense. It's more like the "ending point". But again, this isn't relevant to the change in this PR. I suggest a new issue/PR to discuss improving the wording here.